products/slackware.md

```diff
@@ -0,0 +1,109 @@
+---
+permalink: /slackware
+alternate_urls:
+-   /slackwarelinux
+title: Slackware Linux
+category: os
+releasePolicyLink: http://www.slackware.com/changelog/
+iconSlug: slackware
+changelogTemplate: http://mirrors.slackware.com/slackware/slackware-__RELEASE_CYCLE__/ChangeLog.txt
+activeSupportColumn: false
+versionCommand: cat /etc/os-release
+releaseColumn: false
+releaseDateColumn: true
+realseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
+releasePolicyLink: http://www.slackware.com/faq/do_faq.php?faq=general#4
+purls:
+-   purl: pkg:os/slackwarelinux
+auto:
+-   distrowatch: slackware
+    regex: '^Distribution Release: Slackware Linux (?P<major>\d)\.(?P<minor>\d)$'
+releases:
+-   releaseCycle: "current"
+    codename: "-current"
+    eol: false
+    link: http://www.slackware.com/changelog/current.php?cpu=i386
+    releaseDate: 1993-07-27
+-   releaseCycle: "15.0"
+    codename: "stable"
+    eol: false
+    link: http://www.slackware.com/announce/15.0.php
+    releaseDate: 2022-02-02
+-   releaseCycle: "14.2"
+    eol: false
+    link: http://www.slackware.com/announce/14.2.php
+    releaseDate: 2016-06-30
+-   releaseCycle: "14.1"
+    eol: false
+    link: http://www.slackware.com/announce/14.1.php
+    releaseDate: 2013-11-04
+-   releaseCycle: "14.0"
+    eol: false
+    link: http://www.slackware.com/announce/14.0.php
+    releaseDate: 20l2-09-26
+-   releaseCycle: "13.37"
+    eol: 2018-07-05
+    link: http://www.slackware.com/announce/13.37.php
+    releaseDate: 2011-04-25
+-   releaseCycle: "13.1"
+    eol: 2018-07-05
+    link: http://www.slackware.com/announce/13.1.php
+    releaseDate: 2010-05-19
+-   releaseCycle: "13.0"
+    eol: 2018-07-05
+    link: http://www.slackware.com/announce/13.0.php
+    releaseDate: 2009-08-26
+-   releaseCycle: "12.2"
+    eol: 2013-12-09
+    link: http://www.slackware.com/announce/12.2.php
+    releaseDate: 2008-12-09
+-   releaseCycle: "12.1"
+    eol: 2013-12-09
+    link: http://www.slackware.com/announce/12.1.php
+    releaseDate: 2008-05-01
+-   releaseCycle: "12.0"
+    eol: 2012-08-01
+    link: http://www.slackware.com/announce/12.0.php
+    releaseDate: 2007-07-01
+-   releaseCycle: "11.0"
```

Comment:
If there is a legitimate reason to keep these (they are really old releases!), lets remove `link` from all.

Reply:
@captn3m0 The reason I wanted to show old releases was to give users a sense of what a (typical) support lifecycle looks like for a Slackware stable release (since they often vary in time between release and EOL).  But I'm happy to remove them if you'd like.

Reply:
@alexkoster i think he just means the `link` tags which are not shown when they reach eol dates
so adding those a bit useless plus almost half of them pointing wrong version.
One more thing when we use changelogTemplate thing we only add `link` tag if that release doesnt fit the changelogTemplate's one so as far as i can see you added link tag for each release which is not needed if you have
a working changelogTemplate

Reply:
Removed incorrect `link` from old releases. I like the announce links better:
- http://www.slackware.com/announce/15.0.php vs https://mirrors.slackware.com/slackware/slackware-15.0/ChangeLog.txt

The former is definitely better. Going to update the changelogTemplate for the former and use that wherever possible.